hi-ml-cpath/testhisto/testhisto/mocks/container.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, tmp_path: Path, **kwargs: Any) -> None:
             # declared in TrainerParams:
             max_epochs=2,
             crossval_count=1,
+            ssl_checkpoint_run_id="",
```

Comment:
Why do we need to reset it to the default value here?

Reply:
Good catch - removed